config/locales/exceptions.en.yml

```diff
@@ -139,10 +214,11 @@ en:
         <<: *validation_errors
         code: 'RX157'
         detail: 'Email address is invalid'
-      SM152:
-        <<: *validation_errors
-        code: 'SM152'
-        detail: 'Email address is invalid'
+      MHVAcctCreation155: # Account is already Upgraded
```

Comment:
Not critical since we catch this exception based on the code in the original body, but if it's going to be in this locales file we might as well get the resolution right.